apps/server-web/src/renderer/pages/Server.tsx

```
@@ -95,81 +72,80 @@ export function ServerPage() {
+                    Start Server
+                  </button>
+                )}
               </div>
-            </details>
+            </div>
+
+            <div className="bg-gray-200 dark:bg-[#25272D] border border-gray-200 dark:border-gray-600 rounded-xl overflow-y-auto max-h-[80vh]">
+              <ul className="font-mono text-sm">
+                {
+                  logs && logs.length > 0 && logs.map((log) => (
+                    <li className="flex flex-col sm:flex-row gap-2 sm:gap-6 p-4 transition-colors">
```

Biome (2.4.4)

[error] Missing key property for this element in iterable.

(lint/correctness/useJsxKeyInIterable)